src/modules/home/home.page.tsx

```diff
@@ -39,45 +44,54 @@ export const HomePage = () => {
     setPage(page + 1);
   };
 
+  const onRedirect = (id: string) => {
+    navigation(`/Mais-informações/${id}`);
```

Comment:
nunca usamos maiúsculas na navegação
```suggestion
    navigation(`/mais-informações/${id}`);
```

Reply:
você consegue imaginar o porquê?

Reply:
não utilizando por ela ser um url?

Reply:
e o que isso implica?

Reply:
desconfiguração? como assim?

Reply:
5 motivos:
- Inconsistência, sendo uma parte fundamental da web, sendo padronizada para letras minúsculas. 
- Sensibilidade, alguns servidores são sensíveis a variações podendo ter tratamentos diferentes, que pode levar em erros.
-  SEO, mecanismos de buscar geralmente tratam URLS, tratam como idênticas, porém é recomenadado mater minúsculas.
-  Como disse antes a usabilidade letras minúsculas facilita muito o compartilhamento.